java/sage/plugin/CorePluginManager.java

```diff
@@ -2182,14 +2066,7 @@ else if ("FanArt".equals(packages[i].type))
     }
     finally
     {
-      if (stagedRenameWriter != null)
-      {
-        try
-        {
-          stagedRenameWriter.close();
-        }
-        catch (Exception e){}
-      }
+      IOUtils.closeQuietly(stagedRenameWriter);
```

Comment:
Shouldn't this call get() on the singleton and then close that instead? Otherwise I don't think this is actually going to close that writer ever.